azkaban-spi/src/main/java/azkaban/spi/AzkabanEventReporter.java

```diff
@@ -0,0 +1,19 @@
+package azkaban.spi;
+
+import java.util.Map;
+
+/**
+ * Implement this interface to report flow and job events. Event reporter
+ * can be turned on by setting the property 'azkaban.event.reporting.enabled' to true.
```

Comment:
Refer to the constant instead using javadoc syntax. https://stackoverflow.com/questions/17496038/javadoc-link-to-method-in-other-class ?

Reply:
I have referred to the constants and added `@code` marker. It's not possible to link to classes outside of a module using `@link` or `@see`.

Reply:
Are you sure? I was able to use e.g. 
{@link azkaban.utils.AbstractMailer}